Update user guide, user info, developer guide

LolfoollorS opened the pull request:
Supercede previous PR

LolfoollorS committed:
Update glossary in DeveloperGuide.md
Add placeholder image in docs images

LolfoollorS committed:
Merge branch 'add-zhanfeng-info'

LolfoollorS committed:
Update UserGuide.md

xhphoong commented on docs/DeveloperGuide.md:
```diff
@@ -320,6 +320,10 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
+* **Medical Administrator**: A person who oversees, plan, direct, and coordinate health care delivery.
+* **Patients**: A person receiving or registered to receive medical treatment.
+* **Nurses**: A person trained to care for the sick or infirm, especially in a hospital.
+* **HealthCare Xpress**: A desktop app for managing patients that require home-visits.
```
Maybe the 'c' here should be consistent. should be small 'c' I guess? cuz the readme.md there is small 'c' for Healthcare Xpress

xhphoong commented on docs/UserGuide.md:
```diff
@@ -87,6 +87,19 @@ Examples:
+
+Format:`add c/N n/NAME p/PHONE_NUMBER e/EMAIL g/GENDER [t/TAG]`
+
+> 💡 Tip: A nurse can have any number of tags (including 0).
+> You may type it in any order.
+> To add a nurse, type c/N specifically.
```
Maybe for here can make it as point form and list it down instead of 3 tips is in 1 continuous line.

xhphoong reviewed (commented):
Overall good just a bit minor changes maybe needed?

ou4jairmeiogusrjejiepr reviewed (commented):
LGTM, can just implement xh's suggestions and merge, thanks

johnbenedictyan commented on docs/DeveloperGuide.md:
```diff
@@ -320,6 +320,10 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 ### Glossary
 
+* **Medical Administrator**: A person who oversees, plan, direct, and coordinate health care delivery.
+* **Patients**: A person receiving or registered to receive medical treatment.
+* **Nurses**: A person trained to care for the sick or infirm, especially in a hospital.
```
Do we think that patients and nurses are universally known terms? Maybe we can further contextualise the meaning of patients to those are require home visits?

johnbenedictyan reviewed (commented):
LGTM other than the terms patients and nurses

xhphoong committed:
Update DeveloperGuide.md

Change the patient, nurse, medical administrator definition more specific

xhphoong committed:
Update DeveloperGuide.md

xhphoong committed:
Update UserGuide.md

Minor changes to the format and update the command summary as well as quick start

xhphoong committed:
Update UserGuide.md

Fix mistake

xhphoong committed:
Update UserGuide.md

Fix bulb image

xhphoong committed:
Merge branch 'master' into add-user-guide
